mmv1/products/compute/ResourcePolicy.yaml

```diff
@@ -301,6 +297,7 @@ properties:
       - instance_schedule_policy
       - snapshot_schedule_policy
       - disk_consistency_group_policy
+      - workload_policy
```

Comment:
Will the API return error if both `groupPlacementPolicy` and `workloadPolicy` are provided in the API request?

Reply:
It doesn't work at all for now:
```
Error: Error creating ResourcePolicy: googleapi: Error 400: Invalid value for field 'resource': '{  "name": "tf-test-gce-policy"}'. A ResourcePolicy must have exactly one of [VmMaintenancePolicy, SnapshotSchedulePolicy, GroupPlacementPolicy, InstanceSchedulePolicy, DiskConsistencyGroupPolicy, DataProtectionPolicy, WorkloadPolicy]., invalid
```

But when this issue will be fixed then maybe both `groupPlacementPolicy` and `workloadPolicy` are able to be set in the same time. If so then I will remove that.